src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -205,24 +207,23 @@ export class GcpCreateServerApp extends LitElement {
   }
 
   private renderBillingAccountSetup() {
+    const openLink = '<a href="https://console.cloud.google.com/billing">';
+    const closeLink = '</a>';
     return html`
       <outline-step-view id="billingAccountSetup" display-action="">
         <span slot="step-title">${this.localize('gcp-billing-title')}</span>
-        <span slot="step-description">${this.localize('gcp-billing-description')}</span>
+        <span slot="step-description">
+          ${unsafeHTML(this.localize('gcp-billing-description', 'openLink', openLink, 'closeLink', closeLink))}
+        </span>
         <span slot="step-action">
-          <paper-button id="openBillingPage" @tap="${this.openBillingPage}">
+          <paper-button id="billingPageAction" @tap="${this.handleBillingVerificationNextTap}">
             ${this.localize('gcp-billing-action')}
           </paper-button>
         </span>
         <paper-card class="card">
           <div class="container">
             <img src="images/do_oauth_billing.svg">
-            <p>${this.localize('gcp-billing-body')}</p>
-            <span slot="step-action">
-              <paper-button id="refreshBillingAccounts" @tap="${this.refreshBillingAccounts}">
-                ${this.localize('gcp-billing-refresh')}
-              </paper-button>
-            </span>
+            <p>${unsafeHTML(this.localize('gcp-billing-body', 'openLink', openLink, 'closeLink', closeLink))}</p>
```

Comment:
I think you can set `innerHTML` instead, which may be slightly safer, since it can't leak beyond the element. It would also be consistent with what we do in other parts of the code.

Reply:
I tried this, but it doesn't work.  The lit-html template magic is fully safe, so it escapes any html tags that appear in interpolated values.  The hyperlink tag just appears as text in the UI.